app/src/main/java/agency/five/tmdb/ui/components/MovieCard.kt

```diff
@@ -15,27 +15,37 @@ import androidx.compose.ui.layout.ContentScale
 import androidx.compose.ui.res.dimensionResource
 import androidx.compose.ui.res.painterResource
 import androidx.compose.ui.tooling.preview.Preview
+import androidx.room.ColumnInfo
+import androidx.room.Entity
+import androidx.room.Ignore
+import androidx.room.PrimaryKey
 import coil.compose.rememberImagePainter
 import kotlinx.serialization.Serializable
 
+
+@Entity
 @Serializable
 data class MovieItemViewState(
-    val id: Int = 0,
-    val title: String = "",
-    val overview: String = "Nothing to show",
+    @PrimaryKey(autoGenerate = false)
```

Comment:
You should have separate models for API responses and database data and map into domain models. This will make your architecture more robust.

For now, this is an acceptable solution.